PWGDQ/DataModel/ReducedInfoTables.h

```diff
@@ -40,6 +40,10 @@ namespace reducedevent
 // basic event information
 DECLARE_SOA_COLUMN(Tag, tag, uint64_t);                   //!  Bit-field for storing event information (e.g. high level info, cut decisions)
 DECLARE_SOA_COLUMN(TriggerAlias, triggerAlias, uint32_t); //!  Trigger aliases bit field
+DECLARE_SOA_COLUMN(Q2X, Q2x, float);
+DECLARE_SOA_COLUMN(Q2Y, Q2y, float);
```

Comment:
Please change "Q2x" to "q2x" (naming convention). Same for Q2y. Also, add a comment after each COLUMN definition, similar to the other columns previously defined. This is being used to generate automatic documentation.